lib/simulator-xcode-6.js

```diff
@@ -951,6 +951,68 @@ class SimulatorXcode6 extends EventEmitter {
     `);
   }
 
+  //region Keychains Interaction
+  /**
+   * Create the backup of keychains folder.
+   * The previously created backup will be automatically
+   * deleted if this method was called twice in a row without
+   * `restoreKeychains` being invoked.
+   *
+   * @returns {boolean} True if the backup operation was successfull.
+   */
+  async backupKeychains () {
+    if (!await fs.exists(this.keychainPath)) {
+      return false;
+    }
+
+    const backupPath = await tempDir.path({
+      prefix: `keychains_backup_${Math.floor((1 + Math.random()) * 0x10000).toString(16).substring(1)}`,
+      suffix: '.zip',
+    });
+    const cmdLine = ['-r', backupPath, `${this.keychainPath}${path.sep}`];
+    log.debug(`Creating keychains backup with 'zip ${cmdLine.join(' ')}' command`);
+    await exec('zip', cmdLine);
+    if (_.isString(this._keychainsBackupPath) && await fs.exists(this._keychainsBackupPath)) {
+      await fs.unlink(this._keychainsBackupPath);
```

Comment:
why use `unlink` here and `rimraf` below? is there a perf difference?

Reply:
will change to unlink, since it is anyway a single file